Source/DataSources/CheckerboardMaterialProperty.js

```diff
@@ -16,9 +16,9 @@ var defaultRepeat = new Cartesian2(2.0, 2.0);
  * @constructor
  *
  * @param {Object} [options] Object with the following properties:
- * @param {Property} [options.evenColor=Color.WHITE] A Property specifying the first {@link Color}.
- * @param {Property} [options.oddColor=Color.BLACK] A Property specifying the second {@link Color}.
- * @param {Property} [options.repeat=new Cartesian2(2.0, 2.0)] A {@link Cartesian2} Property specifying how many times the tiles repeat in each direction.
+ * @param {Property|Color} [options.evenColor=Color.WHITE] A Property specifying the first {@link Color}.
```

Comment:
So do these all specifically need `|undefined` as well?  I'm still getting spun up on my TS best practices.

Reply:
Looks like the answer is no, is `undefined` only needed for return types? Or is this because it's an optional parameter anyway so `undefined` is okay?

Reply:
`|undefined` isn't needed on optional properties/parameters.

Reply:
Cool, thanks for confirming.